src/pages/about.page.ts

```diff
@@ -21,27 +21,44 @@ import {NgForm} from '@angular/forms';
   template: `
     <form (ngSubmit)="onSubmit(form.value)" #form="ngForm">
       <label>Character Name:</label>
-      <input 
-        type="text" 
-        name="name" 
-        #characterModel="ngModel" 
+      <input
+        type="text"
+        name="name"
+        #characterModel="ngModel"
         [(ngModel)]="characterForm.name">
       <fieldset ngModelGroup="bioSummary">
         <label>Age:</label>
-        <input 
-          type="number" 
-          name="age" 
+        <input
+          type="number"
+          name="age"
           [(ngModel)]="characterForm.bioSummary.age">
         <label>Size:</label>
-        <input 
-          type="text" 
+        <input
+          type="text"
           name="size"
           [(ngModel)]="characterForm.bioSummary.size">
+        <label>Alignment:</label>
+        <select
+          name="alignment"
+          [(ngModel)]="characterForm.bioSummary.alignment">
```

Comment:
Is this being persisted on the state? I think you should dispatch an action
Even if angular is writing this in the state it would be bad having it writing there directly without action - red flow